tests/integ/modin/frame/test_all_any.py

```diff
@@ -264,26 +257,14 @@ def test_any_bool_only(data, axis):
     ],
 )
 @pytest.mark.parametrize("skipna", [True, False])
-def test_all_float_fallback(data, axis, skipna):
-    # Because axis=None calls the method with axis=0 twice, it incurs an extra query
-    # to check the length of the index after the first call is handled by a fallback
-    with SqlCounter(
-        query_count=9 if axis is None else 8, fallback_count=1, sproc_count=1
```

Comment:
I see 33 occurrences of removed `fallback_count=1` in this PR. In the codebase, however, I see 112 occurrences. I'm assuming the removal of the remaining ones will be covered by the rest of the jira tickets, correct?

Reply:
Right, there are are other JIRAs to remove fallback from datatime accessors, string accessors and groupby APIs.